front/src/App/OpenSpace/TrackWithTalks.js

```diff
@@ -3,11 +3,16 @@ import TalksGrid from './TalksGrid';
 import React from 'react';
 import * as PropTypes from 'prop-types';
 
-export function TrackWithTalks({ track }) {
+export function TrackWithTalks({ talks, reloadTalks, track }) {
+  const filteredTalks = talks.filter((talk) => talk.track.id === track.id);
```

Comment:
Aprovecho y tiro sugerencia. Super opcional extraer la lambda, pero yo a veces lo hago para dar semantica acerca de los filtros.

```suggestion
  const byTrack = (talk) => talk.track.id === track.id;
  const talksFilteredByTrack = talks.filter(byTrack);
```